src/sdk/base/HttpRpcClient.ts

```diff
@@ -58,6 +59,19 @@ export class HttpRpcClient {
     ]);
   }
 
+  async getSkandhaGasPrice(): Promise<Gas> {
+    try {
+      const { maxFeePerGas, maxPriorityFeePerGas } = await this.userOpJsonRpcProvider.send('skandha_getGasPrice', []);
```

Comment:
There is an endpoint to determine the bundler's version/implementation. You can use that to test if the bundler is skandha and then call the appropriate function instead.

Reply:
The method for this is `web3_clientVersion` (so the request would look like this - `await this.userOpJsonRpcProvider.send('web3_clientVersion', []);`).
If the result has the prefix "skandha", then it is our bundler, but I'd suggest that we still keep the fallback to an old method of fetching gas price in case if `skandha_getGasPrice` goes wrong

![image](https://github.com/etherspot/etherspot-prime-sdk/assets/7760339/0d9f0ada-eac5-4420-be8e-e13882d38efe)

Reply:
Done